assets/js/screens/Project/List.jsx

```diff
@@ -40,13 +43,18 @@ const ScreensProjectList = () => {
                 </PrimaryButton> */}
               </div>
             </div>
-            <div className="field">
-              <input
+            <UIFormFieldAndDisplay childClass="has-icons-left">
```

Comment:
Can you explain `childClass` here?  What it's doing?

Reply:
When we use `FontAwesomeIcon` like for search input, the class `has-icons-left` needs to go on the div with `control` class. I tried adding this class manually for the inputs that have icons but it doesn't render inside the input box. To accommodate that, I added a `childClass` prop.

Reply:
Ah, gotcha.  I'm fine with it as-is, or if you'd like to make it a bit more specific, like "controlClass" I wouldn't complain either.